packages/theme/src/launcher.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2018-2020 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Launcher, ILauncher } from '@jupyterlab/launcher';
+
+import { each } from '@lumino/algorithm';
+
+import * as React from 'react';
+
+/**
+ * The known categories of launcher items and their default ordering.
+ */
+const KNOWN_CATEGORIES = ['Notebook', 'Console', 'Elyra', 'Other'];
```

Comment:
how about making `Elyra` the second category? any reason it shouldn't immediately follow the Notebook category?

Reply:
I thought about even disabling the `Console`  category, but I am not sure how popular it is. The only reason they are together (Notebook and Console) is because they have a 1:1 relationship in terms of cards/tiles.

Reply:
I agree `Console` doesn't appear very useful to me.  I suppose it might provide a better "scratchpad" kind of experience.  I think moving `Elyra` up one slot would be good - although when there are lots of kernel types (which typically won't be the case) it could look a little odd since `Elyra` will break the 1:1 parallelism that folks are used to.  (But that shouldn't be a reason for making the Launcher more useful.)